vars/buildPlugin.groovy

```diff
@@ -68,11 +69,23 @@ def call(Map params = [:]) {
                             if (incrementals) { // Incrementals needs 'git status -s' to be empty at start of job
                                 if (isUnix()) {
                                     sh(script: 'git clean -xffd > /dev/null 2>&1',
-                                       label:'Clean for incrementals',
+                                       label: 'Clean for incrementals',
                                        returnStatus: true) // Ignore failure if CLI git is not available
                                 } else {
                                     bat(script: 'git clean -xffd 1> nul 2>&1',
-                                        label:'Clean for incrementals',
+                                        label: 'Clean for incrementals',
+                                        returnStatus: true) // Ignore failure if CLI git is not available
```

Comment:
BTW this is a poor idiom to copy. The Unix version should have been simply

```groovy
sh 'git clean -xffd > /dev/null 2>&1 || :'
```

or

```groovy
sh 'if which git; then git clean -xffd > /dev/null 2>&1; fi'
```

etc.

Reply:
@jglick Not sure about the idiom for the bat command line though, do you have some experience with it?

Reply:
(for info: https://github.com/jenkins-infra/pipeline-library/pull/344)

Reply:
self answer for bat: https://stackoverflow.com/questions/22046780/whats-the-windows-command-shell-equivalent-of-bashs-true-command